database/migrations/2021_08_15_090027_create_projects_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProjectsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('projects', function (Blueprint $table) {
+            $table->id();
+            $table->string('title')->index();
+            $table->text('description')->nullable();
+            $table->dateTime('due_date')->index();
+            $table->foreignId('user_id')->constrained('users');
```

Comment:
@faxunil if the name is with convention, like `xxxxx_id` then you can just do `constrained()` without table name